src/node/SetupServerApi.ts

```diff
@@ -47,48 +48,22 @@ export class SetupServerApi
     this.init()
   }
 
+  private createContext(): SetupServerInternalContext {
+    return {
+      get nodeEvents() {
+        return import('node:events')
```

Comment:
I _think_ this Promise should only ever fire once. I'm not sure if I'm structuring this correctly but the intention is to try to import this module _once_ so each outgoing request (request listener) doesn't pend on this import, increasing the response time).